mkdocs.yml

```diff
@@ -252,6 +252,7 @@ nav:
               - 'Device boot measure': unified-test-documentation/dasharo-performance/403-device-boot-measure.md
               - 'CPU temperature measure': unified-test-documentation/dasharo-performance/401-cpu-temperature.md
               - 'CPU frequency measure': unified-test-documentation/dasharo-performance/402-cpu-frequency.md
+              - 'Stability': unified-test-documentation/dasharo-performance/404-stability.md
```

Comment:
@PrzemyslawBanasiak13 These tests are prepared for the specific platform, so please add also a link in the test matrix.